oidc-ui/Dockerfile

```
@@ -0,0 +1,133 @@
+# Set the environment variable as a placeholder for PUBLIC_URL
+ENV PUBLIC_URL=_PUBLIC_URL_
+
+COPY package*.json ./
+RUN npm install
+#Copy the working directory
+COPY . ./
+RUN npm run build
+
+FROM nginx:1.28.0-alpine3.21-slim
+
+RUN apk update \
```

Hadolint (2.14.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)